general/react_dnd.md

```diff
@@ -0,0 +1,170 @@
+# React Dnd
+
+react-dnd, wether is a good and very flexible drag and drop library for react, has one of the worst documentations ever. Anyway this is not intended to be a full replacement of reat-dnd docs but just some guides and tips to use it on a react application via hooks. For full API reference visit https://react-dnd.github.io/react-dnd/
```

Comment:
> wether is a good and very flexible drag and drop library

Spelling/grammar: "whether it's", but I think you actually want something like "although a good and..."

Also, "react" should be "React".

> has one of the worst documentations ever

Grammar: "has some of the worst documentation ever"

Having said that, I don't think we should be making negative comments about a third-party library in publicly in this tone. Can we go for something a little less judgemental? Maybe:

> react-dnd is a powerful and flexible drag and drop library for React, but there are some gaps in its documentation.

Also:

> Anyway this is not intended to be a full replacement of reat-dnd docs

- "Anyway" need a comma after it.
- "reat-dnd" → "react-dnd"

> to use it on a react application via hooks

"react" → "React"